Apps/Playground/Scripts/validation_native.ts

```diff
@@ -1,21 +1,55 @@
+/// <reference path="./testutils.d.ts" />
+/// <reference path="../../node_modules/babylonjs/babylon.module.d.ts" />
+
+import { NativeEngine, ArcRotateCamera, Scene, SceneLoader, Tools, WebRequest } from "babylonjs";
+
+declare const _native: any;
+
+interface TestConfig {
+    title: string;
+    playgroundId: string;
+    sceneFolder?: string;
+    sceneFilename?: string;
+    referenceImage: string;
+    excludedGraphicsApis?: string[];
+    renderCount?: number;
+    errorRatio?: number;
+    onlyVisual?: boolean;
+    excludeFromAutomaticTesting?: boolean;
+    comment?: string;
+    replace?: string;
+    replaceUrl?: string;
+    specificRoot?: string;
+    scriptToRun?: string;
+    functionToCall?: string;
+    rootPath?: string;
+    threshold?: number;
+}
+
+interface ConfigFile {
+    root: string;
+    tests: TestConfig[];
+}
+type compareFunctionType = (test: TestConfig, renderData: Uint8Array, threshold: number, errorRatio: number, referenceImage?: Uint8Array) => boolean;
```

Comment:
nit: maybe should be UpperCamelCase to be consistent?

```suggestion
type CompareFunctionType = (test: TestConfig, renderData: Uint8Array, threshold: number, errorRatio: number, referenceImage?: Uint8Array) => boolean;
```